lib/ansible/modules/cloud/amazon/ecs_ecr.py

```diff
@@ -235,6 +265,49 @@ def delete_repository_policy(self, registry_id, name):
                 return policy
             return None
 
+    def get_lifecycle_policy(self, registry_id, name):
+        try:
+            res = self.ecr.get_lifecycle_policy(
```

Comment:
Needs exception handling

Reply:
Exception handling is below at https://github.com/leedm777/ansible/blob/005692ee64d6d775019691180697d756b837d52c/lib/ansible/modules/cloud/amazon/ecs_ecr.py#L437-L443. What else is needed?

Reply:
Right, and that wasn't at all obvious to me. I genuinely hadn't noticed the exception handling was there. I prefer the exception handling to be incredibly close to the calling code so that it's obvious. That said, given that this is no worse than currently, I won't block on this.